bundles/AdminBundle/Controller/Admin/DataObject/ClassificationstoreController.php

```diff
@@ -127,7 +127,7 @@ public function deleteGroupAction(Request $request): JsonResponse
      */
     public function createGroupAction(Request $request): JsonResponse
     {
-        $name = $request->get('name');
+        $name = htmlspecialchars($request->get('name'));
```

Comment:
Since we remove all Special Chars in the Frontend, we should also remove them here and not use htmlspecialchars.